presto-main/src/main/java/com/facebook/presto/operator/scalar/TDigestFunctions.java

```diff
@@ -86,4 +104,40 @@ public static Slice scaleTDigestDouble(@SqlType("tdigest(double)") Slice input,
         digest.scale(scale);
         return digest.serialize();
     }
+
+    @ScalarFunction(value = "destructure_tdigest", visibility = EXPERIMENTAL)
```

Comment:
Can we have a better name. @tdcmeehan any suggestions?

Reply:
I can't think of anything better.  To me this is destructuring the internal representation into a struct.

Maybe we can add this as an operator instead, and represent this as a cast?

Reply:
@kaikalur Would `CAST` fit the bill here? If you implement this as `cast` operator we can potentially allow user to pick and choose the fields. Though I'm not sure whether the behavior would be consistent with the spec?

Reply:
I'm not an expert on what a user would want here. If we think they would want individual fields, maybe we abstract from the fact that this is a row and give them a function for each of the fields:

TDIGEST_EXTRACT_SUM(tdigest): DOUBLE

etc.